package.json

```diff
@@ -38,6 +38,7 @@
     "handlebars": "3.0.3",
     "jshint-loader": "0.8.3",
     "jshint-stylish": "2.0.1",
+    "npm": "^6.0.1",
```

Comment:
Why npm in package.json?

Reply:
No idea, I didn't change anything regarding packages.